tests/CommunityToolkit.Mvvm.UnitTests/Collections/Test_ReadOnlyObservableGroupedCollection.cs

```diff
@@ -217,8 +198,9 @@ public void AddGroupInSource_ShouldAddGroup(int sourceInitialItemsCount, int exp
 
         Assert.AreEqual(readOnlyGroup.Count, expectedReadOnlyGroupCount);
 
-        Assert.AreEqual(readOnlyGroup.Last().Key, "Add");
-        CollectionAssert.AreEquivalent(readOnlyGroup.Last(), itemsList);
+        // TODO
```

Comment:
Left a TODO here?

Reply:
Huh, weird. Fixed in 48deec9f77f9c6effba4317f42b8f8436da533f8, thanks!